src/ui/public/index_patterns/_calculate_indices.js

```diff
@@ -17,10 +17,10 @@ define(function (require) {
     function compileOptions(pattern, timeFieldName, start, stop) {
       const constraints = {};
       if (start) {
-        constraints.min_value = { gte: moment(start).valueOf() };
+        constraints.max_value = { gte: moment(start).valueOf() };
```

Comment:
Good question - but let's take that to a separate issue since it isn't behavior that was introduced here.